src/core/databinder.js

```diff
@@ -584,7 +584,7 @@ DataBinder.prototype._setNodeValue = function _setNodeValue($node, value) {
         }
 
         //assign value
-        if (_.contains(['INPUT', 'SELECT', 'TEXTAREA'], $node[0].nodeName)) {
+        if (_.includes(['INPUT', 'SELECT', 'TEXTAREA'], $node[0].nodeName)) {
```

Comment:
A better replacement might be vanilla `Array.includes()`. It would ease the work of upgrading lodash v2 to v4, if we end up doing that later.